<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tutanak</class>
 <widget class="QDialog" name="Tutanak">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLineEdit" name="lineEdit_10"/>
    </item>
    <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Tarih :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonKayit">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLineEdit" name="lineEdit_11"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButtonYaz">
      <property name="text">
       <string>Yazdır</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Ad Soyad :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonMek">
      <property name="text">
       <string>Malzeme Ekle</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget_tutanak">
      <column>
       <property name="text">
        <string>Malzeme Adı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Miktarı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Seçim</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>pushButtonMek</tabstop>
  <tabstop>pushButtonKayit</tabstop>
  <tabstop>pushButtonYaz</tabstop>
  <tabstop>tableWidget_tutanak</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
